hibernate-core/src/test/java/org/hibernate/orm/test/mapping/access/ElementCollectionHHH13721Test.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.orm.test.mapping.access;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Embeddable;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.JiraKey;
+import org.hibernate.testing.orm.junit.SessionFactory;
+import org.hibernate.testing.orm.junit.SessionFactoryScope;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@JiraKey(value = "HHH-13721")
+@DomainModel(
+		annotatedClasses = {
+				ElementCollectionHHH13721Test.Author.class, ElementCollectionHHH13721Test.Book.class
+		}
+)
+@SessionFactory
+class ElementCollectionHHH13721Test {
```

Comment:
Please let's use more significant test class names:
```suggestion
public class ElementCollectionInOneToManyTest {
```